packages/styles/section-header.css

```diff
@@ -0,0 +1,55 @@
+:root {
+  --section-header-heading-font-weight: var(--font-weight-medium, 500);
+  --section-header-description-font-size: var(--text-size-base: 16px);
+  --section-header-description-color: var(--gray-60);
+  --section-header-heading-color: var(--gray-90);
+}
+
+.cauldron--theme-dark {
+  --section-header-description-color: var(--accent-light);
+  --section-header-heading-color: var(--white);
```

Comment:
Can we switch this to `#fff` instead? I got feedback from Jason before that we want to use that since it's shorter and phase out that `--white` var.